eclipse-collections-code-generator/src/main/resources/impl/stack/mutable/synchronizedPrimitiveStack.stg

```diff
@@ -419,8 +419,8 @@ public class Synchronized<name>Stack
     @Override
     public Mutable<name>Stack asUnmodifiable()
     {
-        return new Unmodifiable<name>Stack(this);
-    }
+            return new Unmodifiable<name>Stack(this);
+        }
```

Comment:
It seems the style is broken here. Pls fix.

Reply:
Thx, will fix and commit.